content/docs/command-reference/remote/modify.md

```diff
@@ -108,7 +108,21 @@ these settings, you could use the following options:
   $ dvc remote modify myremote endpointurl https://myendpoint.com
   ```
 
-- `url` - remote location URL
+- `access_key_id` - AWS Access Key ID. May be used (along with
+  `secret_access_key`) instead of `credentialpath`:
+
+  ```dvc
+  $ dvc remote modify myremote access_key_id my-access-key-id
```

Comment:
it's better to use `--local` as we do for HTTP for example to emphasize that these secrets should never go into Git

Reply:
looks like some other remotes should be fixed as well

Reply:
OK. Updating in #1607